### PR TITLE
docs: adding region comments to scope the examples

### PR DESCRIPTION
**Pull request by tarynlucas**
these will be used when we publishing the corresponding how-to on cgc so that we can keep the focus on specific parts of the sample

### CODE REVIEW

**Comment by janetkuo** [comment]
To fix the snippet-bot check error `There are 11 possible violations for not having product prefix`, you might need to check the prefix to `anthosconfig_`

File: fleet-tenancy/terraform/1-fleet/variables.tf [hunk 1]
```diff
@@ -14,8 +14,24 @@
  * limitations under the License.
  */
 
+# [START config_sync_fleet_variables]
 variable "project" {
   type = string
   description = "Name of GCP project to host the Fleet"
 }
 
+variable "gcp_sa_id" {
```
Comment (nan-yu) [inline fleet-tenancy/terraform/1-fleet/variables.tf:23]: These variables are removed by https://github.com/GoogleCloudPlatform/anthos-config-management-samples/pull/104. You need to rebase your commit.

File: fleet-tenancy/terraform/2-fleet-default-config/variables.tf [hunk 2]
```diff
@@ -14,8 +14,14 @@
  * limitations under the License.
  */
 
+# [START config_sync_fleet_default_variables]
 variable "project" {
   type = string
   description = "Name of GCP project"
 }
 
+variable "sa_key_file" {
```
Comment (nan-yu) [inline fleet-tenancy/terraform/2-fleet-default-config/variables.tf:23]: This is removed.

File: fleet-tenancy/terraform/3-clusters/variables.tf [hunk 3]
```diff
@@ -14,8 +14,14 @@
  * limitations under the License.
  */
 
+# [START config_sync_cluster_variables]
 variable "project" {
   type = string
   description = "Name of GCP project"
 }
 
+variable "sa_key_file" {
```
Comment (nan-yu) [inline fleet-tenancy/terraform/3-clusters/variables.tf:23]: ditto, need to rebase.

File: fleet-tenancy/terraform/4-scopes-and-namespaces/variables.tf [hunk 4]
```diff
@@ -14,8 +14,14 @@
  * limitations under the License.
  */
 
+# [START config_sync_scope_variables]
 variable "project" {
   type = string
   description = "Name of GCP project"
 }
 
+variable "sa_key_file" {
```
Comment (nan-yu) [inline fleet-tenancy/terraform/4-scopes-and-namespaces/variables.tf:23]: ditto

**Review by nan-yu — APPROVED** [review]
/lgtm

File: .DS_Store
Comment (janetkuo) [inline .DS_Store:1]: These .DS_Store files are accidentally pushed and should be removed.

**Pull request merged**